tests/torch/data/reference_graphs/quantized/autoq/inception_v3_device_VPU__eval_subset_ratio_0.4__with_ignored_scopes.dot

```diff
@@ -1,552 +1,553 @@
 strict digraph  {
-"0 /nncf_model_input_0" [id=0, label="nncf_model_input_#0", scope="", style=filled, type=nncf_model_input];
-"1 SymmetricQuantizer/symmetric_quantize_0" [color=red, id=1, label="AFQ_[B:4 M:S SGN:S PC:N]_#1_G0", scope=SymmetricQuantizer, style=filled, type=symmetric_quantize];
```

Comment:
it's strange that reference graphs have significantly changed after this refactoring.
bitwidth assignment is different, so HAWQ and AutoQ algorithms started to work differently. Does it mean that we are breaking some old behaviour? 

for HAWQ it's relevant for inception-v3 with target_scopes only, but almost all AutoQ graphs were heavily modified

Reply:
I removed some crud from one of the automl files:
https://github.com/openvinotoolkit/nncf/pull/735/files#diff-748c64221b0143c3f76c4905d55c0389e952c261860051ad3081989170034f3aR285

and one of the things I changed was the indexing of quantizers in a table; now the indices do not have the ID of the quantizer node in the compressed dynamic graph (which I think the AutoML code shouldn't really depend on and use) prepended and the quantizers in a table are therefore sorted differently - based on the quantizer ID (which is a target node name + pre/post/weight quantization type specifier). 
Ordered quantizer table indices before:
```
1-/nncf_model_input_0|OUTPUT
2-Inception3/BasicConv2d[Conv2d_1a_3x3]/NNCFConv2d[conv]module_weight
6-Inception3/BasicConv2d[Conv2d_1a_3x3]/RELU_0|OUTPUT
10-Inception3/BasicConv2d[Conv2d_2a_3x3]/RELU_0|OUTPUT
11-Inception3/BasicConv2d[Conv2d_2b_3x3]/NNCFConv2d[conv]module_weight
15-Inception3/BasicConv2d[Conv2d_2b_3x3]/RELU_0|OUTPUT
17-Inception3/BasicConv2d[Conv2d_3b_1x1]/NNCFConv2d[conv]module_weight
21-Inception3/BasicConv2d[Conv2d_3b_1x1]/RELU_0|OUTPUT
22-Inception3/BasicConv2d[Conv2d_4a_3x3]/NNCFConv2d[conv]module_weight
27-Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch1x1]/NNCFConv2d[conv]module_weight
28-Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch1x1]/NNCFConv2d[conv]/conv2d_0|INPUT0
28-Inception3/InceptionA[Mixed_5b]/avg_pool2d_0|INPUT0                                       # !!!
28-Inception3/InceptionA[Mixed_5b]/avg_pool2d_0|OUTPUT                                       # !!!                  
32-Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch5x5_1]/NNCFConv2d[conv]module_weight
33-Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch5x5_1]/NNCFConv2d[conv]/conv2d_0|INPUT0
33-Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch5x5_1]/RELU_0|OUTPUT
...
```

After:
```
/nncf_model_input_0|OUTPUT
Inception3/AdaptiveAvgPool2d[avgpool]/adaptive_avg_pool2d_0|OUTPUT             # !!!
Inception3/BasicConv2d[Conv2d_1a_3x3]/NNCFConv2d[conv]/conv2d_0|WEIGHT
Inception3/BasicConv2d[Conv2d_1a_3x3]/RELU_0|OUTPUT
Inception3/BasicConv2d[Conv2d_2a_3x3]/RELU_0|OUTPUT
Inception3/BasicConv2d[Conv2d_2b_3x3]/NNCFConv2d[conv]/conv2d_0|WEIGHT
Inception3/BasicConv2d[Conv2d_2b_3x3]/RELU_0|OUTPUT
Inception3/BasicConv2d[Conv2d_3b_1x1]/NNCFConv2d[conv]/conv2d_0|WEIGHT
Inception3/BasicConv2d[Conv2d_3b_1x1]/RELU_0|OUTPUT
Inception3/BasicConv2d[Conv2d_4a_3x3]/NNCFConv2d[conv]/conv2d_0|WEIGHT
Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch1x1]/NNCFConv2d[conv]/conv2d_0|INPUT0
Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch1x1]/NNCFConv2d[conv]/conv2d_0|WEIGHT
Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_1]/NNCFConv2d[conv]/conv2d_0|INPUT0
Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_1]/NNCFConv2d[conv]/conv2d_0|WEIGHT
Inception3/InceptionA[Mixed_5b]/BasicConv2d[branch3x3dbl_1]/RELU_0|OUTPUT

```
Diffs are highlighted with `# !!!`. Therefore the order in which the agent assigns decisions to quantizers may have changed, and the result has changed as well. If the topological ordering of the quantizers in a table is really essential to correct AutoQ algo operation, I could try to prepend the indices with the IDs of the nodes the quantizers target in an NNCFGraph which I will obtain by matching the `target_node_name` attribute of the `QuantizerId` to the model's original graph, but not to the compressed dynamic graph. I think that the AutoQ should operate correctly on average independent of the order of the quantizers in the search space we pass to it.